code/modules/clothing/shoes/clownshoes.dm

```diff
@@ -0,0 +1,204 @@
+	slowdown = 15
+
+/obj/item/clothing/shoes/clown_shoes/prison/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/chain.ogg'=1), 50)
+
+
+/obj/item/clothing/shoes/clown_shoes/prison/allow_attack_hand_drop(mob/user)
+	if(ishuman(user))
+		var/mob/living/carbon/human/C = user
+		if(C.shoes == src)
+			to_chat(user, "<span class='warning'>You need help taking these off!</span>")
+			return FALSE
+	return ..()
+
+/obj/item/clothing/shoes/clown_shoes/prison/MouseDrop(atom/over)
+	var/mob/m = usr
+	if(ishuman(m))
+		var/mob/living/carbon/human/c = m
+		if(c.shoes == src)
+			to_chat(c, "<span class='warning'>You need help taking these off!</span>")
+			return
+	return ..()
+
+
+/obj/item/clothing/shoes/clown_shoes/engi
+	desc = "These boots were made for working, but that's not what we're gonna do.  Ctrl-click to toggle waddle dampeners."
+	name = "clown boots"
+	icon_state = "clown_engi"
+	inhand_icon_state = "clown_shoes_engi"
+
+/obj/item/clothing/shoes/clown_shoes/engi/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/catwalk3.ogg'=1,'sound/effects/footstep/carpet5.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/hydro
+	desc = "A bit prickly, and somewhat loose. Perhaps you can get a new sole? Ctrl-click to toggle waddle dampeners."
+	name = "clown grass shoes"
+	icon_state = "clown_hydro"
+	inhand_icon_state = "clown_shoes_hydro"
+
+/obj/item/clothing/shoes/clown_shoes/hydro/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/grass2.ogg'=1,'sound/effects/footstep/grass3.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/clogs
+	desc = "Great sole support, but shouldn't these bee yellow? Ctrl-click to toggle waddle dampeners."
+	name = "clown clogs"
+	icon_state = "clown_clogs"
+	inhand_icon_state = "clown_shoes_clogs"
+
+/obj/item/clothing/shoes/clown_shoes/clogs/attackby(obj/H, mob/user, loc)
+	..()
+	if (H.type == /obj/item/reagent_containers/honeycomb || H.type== /obj/item/grown/sunflower)
+		var/location = get_turf(user)
+		new /obj/item/clothing/shoes/clown_shoes/dutch(location)
+		qdel(H)
+		qdel(src)
+	return
+
+/obj/item/clothing/shoes/clown_shoes/clogs/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/footstep/woodclaw1.ogg'=1,'sound/effects/footstep/woodclaw3.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/dutch
+	desc = "Why are dutch clowns not tall? All the big one bounced out the ceiling! TURN ON YOUR WADDLE DAMPENERS WITH CTRL-CLICK FOR GODS SAKE!!"
+	name = "dutch clown clogs"
+	icon_state = "clown_dutch"
+	inhand_icon_state = "clown_shoes_dutch"
+
+/obj/item/clothing/shoes/clown_shoes/dutch/Initialize()
+	. = ..()
+	AddComponent(/datum/component/squeak, list('sound/effects/clownstepdutch.ogg'=1), 100)
+
+
+/obj/item/clothing/shoes/clown_shoes/med
+	desc = "A more sanitary version of the clown shoes.  Ctrl-click to toggle waddle dampeners."
```

Comment:
it'd be nice if medical clown shoes actually had more bio armor than normal or something, but come to think of it, do any medbay shoes have (greater than normal) bio armor?